doc/examples/syn_registration_2d.py

```diff
@@ -91,13 +91,13 @@
 # Now let's warp the moving image and see if it gets similar to the static
 # image
 
-warped_moving = mapping.transform(moving, "linear")
+warped_moving = mapping.transform(moving, interpolation="linear")
 regtools.overlay_images(
     static,
     warped_moving,
     title0="Static",
     title_mid="Overlay",
-    title_1="Warped moving",
+    title1="Warped moving",
```

Comment:
Is this a recent change to the api of this function?

Reply:
no, it was a typo not detected on a previous PR